mdbenchmark/utils.py

```diff
@@ -125,3 +127,17 @@ def guess_ncores():
                        click.style('WARNING', fg='yellow', bold=True)))
         total_cores = 1
     return total_cores
+
+
+
+def detect_md_engine(modulename):
```

Comment:
I talked with @mimischi today. This would be better suited in the mdengine/__init__.py file.